app/views/customers/show.html.erb

```diff
@@ -0,0 +1,23 @@
+<h1><%= @customer.name %></h1>
+
+<h2> Items </h2>
+<ul>
+  <% @customer.items.each do |item| %>
+    <li>
+      Supermarket: <%= item.supermarket.name %>
+      Item: <%= item.name %>
+      Price: <%= item.price %>
+    </li>
+  <% end %>
+</ul>
+
+<h2> Add Item </h2>
+<%= form_with url: "/customers/#{@customer.id}/items", method: :post, local: true do |f| %>
+    <div>
+      <%= label_tag :item_id, "Item ID" %>
+      <%= text_field_tag :item_id %>
+    </div>
+      <%= submit_tag "Add Item" %>
```

Comment:
We don't want to use form_tags here since you're using a form_with. Revisit and review the lesson on forms to see the correct syntax.